farm-22-04/scripts/012-farm.sh

```diff
@@ -0,0 +1,51 @@
+#!/bin/sh
+
+# Create the farm user
+useradd --home-dir /home/farm \
+        --shell /bin/bash \
+        --create-home \
+        --system \
+        farm
+
+# Setup the home directory
+chown -R farm: /home/farm
+
+# Setup
+chown -R farm: /var/www/html
+
+usermod -aG sudo farm
+
+# Create farm folder
+mkdir /home/farm
+cd /home/farm
+
+# Create react application
+npx create-react-app client
+
+# Copy sample react project
+cp /etc/sample-project/src/* /home/farm/client/src
+
+# Create server
+mkdir server
+cd server
+
+pip install fastapi uvicorn[standard] motor gunicorn pipenv
+
+# Copy sample server
+cp /etc/sample-project/server/* /home/farm/server
+
+pipenv install -r requirements.txt
+ufw allow 8000/tcp
+
+# Delete sample project
+rm -r /etc/sample-project
+rm /home/farm/client/src/logo.svg
+
+cd /home/farm/client/src && npm run build
+
+sudo npm install pm2@latest -g --no-optional
+
+su - farm -c "cd /home/farm/server && pm2 start \"gunicorn -w 4 -k uvicorn.workers.UvicornWorker main:app\" --name \"sample_api\""
```

Comment:
So, we are running nginx -> gunicorn -> uvicorn -> main app. 
This is a quite sophisticated setup for a droplet. I will research if it makes sense to use gunicorn here, for now leave it as is